<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QMdiArea" name="mdiArea" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>60</y>
      <width>421</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="actionSaveAs" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCut" />
    <addaction name="actionCopy" />
    <addaction name="actionPaste" />
    <addaction name="separator" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout" />
    <addaction name="actionAboutQt" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew" />
   <addaction name="actionOpen" />
   <addaction name="actionSave" />
   <addaction name="separator" />
   <addaction name="actionCut" />
   <addaction name="actionCopy" />
   <addaction name="actionPaste" />
  </widget>
  <action name="actionNew" >
   <property name="icon" >
    <iconset resource="editor.qrc" >
     <normaloff>:/images/images/new.png</normaloff>:/images/images/new.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;New</string>
   </property>
   <property name="toolTip" >
    <string>New</string>
   </property>
   <property name="statusTip" >
    <string>Create a new file</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="icon" >
    <iconset resource="editor.qrc" >
     <normaloff>:/images/images/open.png</normaloff>:/images/images/open.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open...</string>
   </property>
   <property name="statusTip" >
    <string>Open an existing file</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="icon" >
    <iconset resource="editor.qrc" >
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save</string>
   </property>
   <property name="statusTip" >
    <string>Save the document to disk</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
   <property name="statusTip" >
    <string>Exit the application</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSaveAs" >
   <property name="text" >
    <string>Save &amp;As</string>
   </property>
   <property name="statusTip" >
    <string>Save the document under a new name</string>
   </property>
  </action>
  <action name="actionCut" >
   <property name="icon" >
    <iconset resource="editor.qrc" >
     <normaloff>:/images/images/cut.png</normaloff>:/images/images/cut.png</iconset>
   </property>
   <property name="text" >
    <string>Cu&amp;t</string>
   </property>
   <property name="statusTip" >
    <string>Cut the current selection to the clipboard</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy" >
   <property name="icon" >
    <iconset resource="editor.qrc" >
     <normaloff>:/images/images/copy.png</normaloff>:/images/images/copy.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Copy</string>
   </property>
   <property name="statusTip" >
    <string>Copy the current selection to the clipboard</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste" >
   <property name="icon" >
    <iconset resource="editor.qrc" >
     <normaloff>:/images/images/paste.png</normaloff>:/images/images/paste.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Paste</string>
   </property>
   <property name="statusTip" >
    <string>Paste the clipboard into the current selection</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
   <property name="statusTip" >
    <string>Show this application's About box</string>
   </property>
  </action>
  <action name="actionAboutQt" >
   <property name="text" >
    <string>About &amp;Qt</string>
   </property>
   <property name="statusTip" >
    <string>Show Qt's About box</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="editor.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>368</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open()</slot>
 </slots>
</ui>
